### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by lepelog** [lepelog/sslib#91]
add tests for logic to make sure we don't break things on accident

**Comment by CovenEsme** [comment]
Pretty sure this has been done too

**Comment by lepelog** [comment]
not really, the only thing we test is a very specific set of settings and we just make sure 2 different runs with the same seed+settings produce the same spoiler log

**Comment by CovenEsme** [comment]
#579 is a step towards this
